calculate.js

```diff
@@ -0,0 +1,9 @@
+let Add = (a, b) => { return a + b; };
```

Comment:
1. Number( ... ) 제거
2. 함수는 소문자로
3. 한줄 식이면 가로와 return 생략가능

```suggestion
let add = (a, b) => a + b
```